src/redux/reducers/componentConfig/initialState.js

```diff
@@ -71,7 +71,9 @@ const embeddingContinuousInitialConfig = {
   marker: markerBaseState,
   labels: labelBaseState,
   logEquation: 'datum.expression*1',
-  shownGene: 'notSelected',
+  shownGene: null,
+  expressionValue: 'raw',
+  truncatedValues: true,
```

Comment:
I assume this fixes the capped / uncapped issue but it's not very clear to me how. Also, truncated / capped are equivalent names? it's a bit confusing

Reply:
`expressionValue` and `truncatedValues` were being searched for but didn't exist in the initial config of this plot, so what I did to fix that issue was add them to initial config